lib/cloudfunc.js

```diff
@@ -218,6 +218,16 @@
                     mode        : '--- --- ---'
                 });
             }
+
+            // OSC_CUSTOM_CODE hide all dot files
+            n = files.length;
+            var filesWithoutDots = [];
+            for (i = 0; i < n; i++) {
+                if (files[i].name.charAt(0) !== '.') {
+                    filesWithoutDots.push(files[i]);
+                }
+            }
+            files = filesWithoutDots;
```

Comment:
Use filter method instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter?redirectlocale=en-US&redirectslug=JavaScript%2FReference%2FGlobal_Objects%2FArray%2Ffilter

Also, if arrow functions are enabled or available in this version of node: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Then it could be as simple as:

    files = files.filter(f => f.name.charAt(0) !== '.');